ui/appui-react/src/appui-react/configurableui/InternalConfigurableUiManager.ts

```diff
@@ -26,26 +26,26 @@ import { InternalModalDialogManager } from "../dialog/InternalModalDialogManager
 import { SyncUiEventDispatcher } from "../syncui/SyncUiEventDispatcher";
 
 /** Ui Activity Event Args interface.
- * @internal
+ * @public
```

Comment:
This file is not exported in the barrel, so even if they are marked public, they will not show up.
These events are part of the internal timers and the results of the timers are exposed in events like "onFronstageDeactivatedEvent". Do we really want to expose those ? (If so, we should move these events somewhere else as well).

Reply:
DR uses this in their testing. I'll check in with them.